code/modules/surgery/organs/organ_internal.dm

```diff
@@ -9,6 +9,19 @@
 	var/unremovable = FALSE //Whether it shows up as an option to remove during surgery.
 	/// An associated list of organ datums that this organ has.
 	var/list/datum/organ/organ_datums
+	/// This contains the hidden RnD levels of an organ to prevent rnd from using it.
+	var/hidden_origin_tech
+	/// What is the level of tech for the hidden tech type?
+	var/hidden_tech_level = 1
+	/// How much is this organ worth in the xenobiology organ analyzer?
+	var/analyzer_price = 10
+	/// what quality is this organ? Only useful for xeno organs
+	var/organ_quality = ORGAN_NORMAL
+	/// Does this organ originate from the xenobiology dissection loop?
+	var/is_xeno_organ = FALSE
+	var/cooldown_low = 300
+	var/cooldown_high = 300
+	var/next_activation = 0
 
```

Comment:
Please document what these cooldown variables are in reference to (it's not very clear at a glance that they're related to xeno organs), maybe name it something like `organ_ability_cooldown` or `organ_action_cooldown`?